cmd/operator-opamp-bridge/internal/agent/agent.go

```diff
@@ -138,6 +141,10 @@ func (agent *Agent) generateCollectorPoolHealth() (map[string]*protobufs.Compone
 			Healthy:            isPoolHealthy,
 		}
 	}
+	// TODO: Figure out how to tie this to the agent health from the proxy.
```

Comment:
this will be done in a follow up.

Reply:
yes and no, it's really just more in depth information for the collector pods, you can see how i'll do this in [this branch](https://github.com/open-telemetry/opentelemetry-operator/compare/main...jaronoff97:opentelemetry-operator:instance-id-correlation?expand=1#diff-77162bc58930af5dede9b3b691b114de2790b1ac42e5ab21a0f6a0528630879bR116-R157).